packages/components/button/src/__tests__/Button.spec.tsx

```diff
@@ -1,15 +1,192 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import Button from '../Button';
 import { ThemeProvider } from '@douro-ui/react';
+import { ButtonProps } from '../button.types';
 
-describe('<Button />', () => {
-  it('should render properly', () => {
-    render(
-      <ThemeProvider>
-        <Button label="Foo" />
-      </ThemeProvider>,
+const renderTheBtnType = (
```

Comment:
You should create a reusable custom render to wrap all the providers (e.g. ThemeProvider in this case) and pass the components.

https://testing-library.com/docs/react-testing-library/setup/#custom-render

Reply:
I reviewed the comment, but it seems you might have looked at an older commit. The current commit is already wrapped and uses the correct props.